File: Backend API/Locafest/com.locafest.Domain/UsuarioContext/Entities/Operador.cs
using com.locafest.Domain.UsuarioContext.Enum;
using System;
using System.Collections.Generic;
using System.Text;

namespace com.locafest.Domain.UsuarioContext.Entities
{
    public class Operador : BaseEntity
    {
        public string Matricula { get; set; }
        public string Nome { get; set; }
        public Guid UsuarioId { get; set; }
        public Usuario Usuario { get; set; } 
        public int PerfilId { get { return (int) PerfilEnum.Operador; } }
    }
}

File: Mobile FrontEnd/locafest/lib/app/shared/entities/operador.dart
class Operador {
  String id;
  String matricula;
  String nome;
  String usuarioId;

  Operador({this.id, this.matricula, this.nome, this.usuarioId});

  Operador.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    matricula = json['matricula'];
    nome = json['nome'];
    usuarioId = json['usuarioId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['matricula'] = this.matricula;
    data['nome'] = this.nome;
    data['usuarioId'] = this.usuarioId;
    return data;
  }
}